src/main/scala/org/hammerlab/guacamole/variants/Concordance.scala

```diff
@@ -1,154 +0,0 @@
-package org.hammerlab.guacamole.variants
```

Comment:
opportunistically removed since I didn't think it'd been used in years, has no tests, etc… I've put it back in though